src/stylesheets/screens/survey-list.scss

```diff
@@ -33,6 +32,11 @@
   }
 
   &__navigation {
+    position: relative;
+    bottom: rem(96px);
+
+    float: right;
+
     margin: rem(15px 0 0 auto);
```

Comment:
Should we also change this `rem(34px 0 0 auto)`?
![Screen Shot 2563-11-18 at 11 31 14](https://user-images.githubusercontent.com/1772999/99483810-9ca52f00-2991-11eb-8195-808bea5ea923.png)

Reply:
@rafayet-monon I prefer to an image instead of draw it ourself 😅
What do you think? 😛

Reply:
@carryall Fixed in [1bbdeea](https://github.com/rafayet-monon/surveyor/pull/17/commits/1bbdeeae86507eb1b8ecd22536d5bd0d7a579e3c).

I used `bottom` here so the margin is kind of unnecessary, so removed it. But I can assure that there is a `34px` between there now.

<img width="263" alt="Screenshot 2020-11-19 at 9 22 37 AM" src="https://user-images.githubusercontent.com/14927672/99617798-3c25f880-2a4a-11eb-8fd0-0242a964b303.png">

Reply:
@Lahphim Fixed in [1bbdeea](https://github.com/rafayet-monon/surveyor/pull/17/commits/1bbdeeae86507eb1b8ecd22536d5bd0d7a579e3c).

Reply:
But me and @carryall are thinking about a svg icon 😅
The way to do that is export a black arrow as svg and make it as share icon.
And then we draw a white circle which has that icon inside.
What do you think? 🤔

Reply:
@Lahphim @carryall I have created a separate task for svg icon sitemap - https://github.com/rafayet-monon/surveyor/projects/1#card-49663110
Will work on that after a while.